sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_models.py

```diff
@@ -2,22 +2,22 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
-from typing import TYPE_CHECKING
+from typing import Dict, Generic, Optional, TypeVar, Union
+
+from azure.core.rest import HttpResponse
 
 from ._enums import SettingType
+from ._generated_models import (
+    FullBackupOperation,
+    Permission,
+    RoleAssignment,
+    RoleAssignmentProperties,
+    RoleAssignmentPropertiesWithScope,
+    RoleDefinition,
+    Setting,
+)
 
-if TYPE_CHECKING:
-    from typing import Any, Dict, Union
-    from azure.core.rest import HttpResponse
-    from ._generated_models import (
-        FullBackupOperation,
-        Permission,
-        RoleAssignment,
-        RoleAssignmentProperties,
-        RoleAssignmentPropertiesWithScope,
-        RoleDefinition,
-        Setting,
-    )
+T = TypeVar("T", bool, str)
```

Comment:
I think this should use `bound`, and probably a more descriptive name
```suggestion
SettingValueType = TypeVar("SettingValueType", bound=Union[bool, str])
```